src/components/slider/ColorSliderGroup.js

```diff
@@ -62,18 +71,29 @@ export default class ColorSliderGroup extends PureBaseComponent {
     _.invoke(this.props, 'onValueChange', value);
   }
 
+  getSlider = (type) => {
```

Comment:
I think it's better to rename the  function renderSlider like we usually do

Reply:
Fixed :S